src/general/OptionsMenu.cs

```diff
@@ -477,4 +496,20 @@ private void OnChromaticAberrationValueChanged(float amount)
     {
         Settings.ChromaticAmount = amount;
     }
+
+    private void OnAutoSaveToggled(bool pressed)
+    {
+        Settings.AutoSaveEnabled = pressed;
+        maxAutosaves.Editable = pressed;
+    }
+
+    private void OnMaxAutoSavesValueChanged(float value)
+    {
+        Settings.MaxAutoSaves = (int)value;
+    }
+
+    private void OnMaxQuickValueChanged(float value)
```

Comment:
I think you made a small typo on the function name here, it should be `OnMaxQuickSavesValueChanged` since the SpinBox of MaxQuickSaves `value_changed` signal is connected to that.

Reply:
I think I just got the same error when testing:
```
E 0:00:01.608   emit_signal: Error calling method from signal 'value_changed': 'Control(OptionsMenu.cs)::OnMaxQuickSavesValueChanged': Method not found..
  <C++ lähdekoodi>core/object.cpp:1228 @ emit_signal()
  <Pinojäljitys>:0 @ void Godot.NativeCalls.godot_icall_1_707(IntPtr , IntPtr , System.Double& )()
                Range.cs:277 @ void Godot.Range.SetValue(Double )()
                Range.cs:104 @ void Godot.Range.set_Value(Double )()
                OptionsMenu.cs:216 @ void OptionsMenu.SetSettingsFrom(Settings )()
                MainMenu.cs:122 @ void MainMenu.RunMenuSetup()()
                MainMenu.cs:44 @ void MainMenu._Ready()()

```

Reply:
Yeah that's the error